clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus_test.go

```diff
@@ -787,3 +770,20 @@ func TestAPIResponsivenessCustomThresholds(t *testing.T) {
 		})
 	}
 }
+
+// FIXME: Remove it.
+func TestKlog(t *testing.T) {
+	klog.LogToStderr(false)
```

Comment:
Based on chat with @pohly:
```
Same problem: test/e2e/framework is opinionated and installs a custom logger during init (test/e2e/framework/ginkgologger.go).
The effect is that klog.SetOutput has no effect.
```

Reply:
https://github.com/kubernetes/perf-tests/pull/3051 removed the biggest part.
I will try to finish it and that should hopefully unblock this PR.